sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Generated/Models/SoapApiType.cs

```diff
@@ -45,6 +46,8 @@ public SoapApiType(string value)
         public static SoapApiType GraphQL { get; } = new SoapApiType(GraphQLValue);
         /// <summary> Imports the API having a OData front end. </summary>
         public static SoapApiType OData { get; } = new SoapApiType(ODataValue);
+        /// <summary> Imports the API having a gRPC front end. </summary>
+        public static SoapApiType GRPC { get; } = new SoapApiType(GRPCValue);
```

Comment:
`ContentFormat` uses `Grpc`, better to unify the names.

Reply:
@archerzz arent all your comments for `Generated/Models` 

Wouldn't we need to update the rest spec for fixing this issues?

Reply:
> @archerzz arent all your comments for `Generated/Models`
> 
> Wouldn't we need to update the rest spec for fixing this issues?

You can do this. But it's much harder to change specs, since those are breaking changes.